### PR TITLE
[feature](executor) Add some check when create workload group/workload schedule policy

### PR DESCRIPTION
**Pull request by wangbo**
Add some check when create workload group/workload schedule policy

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -2318,6 +2318,18 @@ public class Config extends ConfigBase {
...
     public static int workload_action_interval_ms = 10000; // 10s
 
+    @ConfField(mutable = true, masterOnly = true)
+    public static int workload_max_policy_num = 25;
+
+    @ConfField(mutable = true, masterOnly = true)
+    public static int workload_max_condition_num_in_policy = 5;
+
+    @ConfField(mutable = true, masterOnly = true)
+    public static int workload_max_action_num_in_policy = 5; // mainly used to limit set session var action
```
Comment (yiguolei) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:2328]: Do we really support more than 1 action in same policy??

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**